src/Views/AppInfoView.vala

```diff
@@ -181,36 +181,38 @@ namespace AppCenter.Views {
             }
 #endif
 
-            var percent_translated = package_component.get_language (
-                // Expects language without locale
-                package_component.get_active_locale ().split ("_")[0]
-            );
-
-            if (percent_translated < 100) {
-                if (percent_translated == -1) {
-                    var locale = new ContentType (
-                        _("May Not Be Translated"),
-                        _("This app does not provide language information"),
-                        "metainfo-locale"
-                    );
-
-                    oars_flowbox.add (locale);
-                } else if (percent_translated == 0) {
-                    var locale = new ContentType (
-                        _("Not Translated"),
-                        _("This app is not available in your language"),
-                        "metainfo-locale"
-                    );
+            if (!package.is_native && !package.is_os_updates && !package.is_runtime_updates) {
```

Comment:
I think filtering out OS and runtime updates pages makes sense while those are still using AppInfoView, but I don't think we should exempt our own packages from these warnings